bigquery/main.go

```diff
@@ -74,7 +74,12 @@ type bqRow struct {
 	Tags           []string
 	Env            []string
 	LogURL         string
-	Substitutions  [][]string
+	Substitutions  []substitution
```

Comment:
Does this need to be a pointer like the other structs?

Reply:
doesn't need to be, but changing for consistency